Resources/Prototypes/Entities/Clothing/Neck/armbands.yml

```diff
@@ -0,0 +1,54 @@
+- type: entity
+  parent: ClothingNeckBase
+  id: ClothingNeckArmbandsCargo
+  name: cargo armband
+  description: An armband showing commitment to supply, cargo, and salvage.
+  components:
+  - type: Sprite
+    sprite: Clothing/Neck/Armbands/cargoband.rsi
+  - type: Clothing
+    sprite: Clothing/Neck/Armbands/cargoband.rsi

```

Comment:
this does not seem like neckwear. i imagine that was just the easiest way to implement it right now? i think i'd prefer if it just wasnt in for now until we can do it properly (attaching to jumpsuits)